cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -4191,5 +4226,107 @@ TEST(TestArrowReadDeltaEncoding, DeltaBinaryPacked) {
 }
 #endif
 
+struct StatisticsTestParam {
```

Comment:
Can you perhaps create a separate `statistics_test.cc`? This test file is getting much too long already.

Reply:
created arrow_statistics_test.cc